### ISSUE TITLE
Problem: Timezone as string not working in config

### ISSUE THREAD

**Issue by deini** [c-rack/quantum-elixir#108]
If I set something like
```elixir
config :quantum,
  cron: [
    "17 22 * * * Integrations.MOTD.generate"
  ],
  timezone: "America/Chicago"
```
I get `** (RuntimeError) Unsupported timezone: America/Chicago`.

Can I only use this type of timezones at runtime?

**Comment by c-rack** [comment]
Currently yes, as it is also described in the README. Any pull requests to extend that are welcome :-)

Could you also please rename this issue according our contribution policy? Thanks.

**Comment by deini** [comment]
@c-rack Can you point me at where in the README this is mentioned? I read over and over again the Timezone section but couldn't find anything about not being able to use it as a string in the config. 🔎 Not sure what I'm missing.

**Comment by c-rack** [comment]
It is a little bit hidden and that may be improved as well, but you find it in the comment to the `timezone` field in the section "Named jobs". 

Anyway, I agree that this is unexpected behavior and should be changed.

**Comment by deini** [comment]
I'll try to submit a PR tonight, just wanted to make sure I was not missing something. Thanks!

**Comment by amay** [comment]
Any updates on this? I'm trying to specify a timezone as a string in a specific job config. I don't get any errors on startup but I'm observing that it is silently using UTC. I see that https://github.com/c-rack/quantum-elixir/pull/90 was supposed to add per-job timezone support, and the README currently indicates that you can specify a timezone for a job via a string here: https://github.com/c-rack/quantum-elixir#timezone. Is the README incorrect?

**Comment by c-rack** [comment]
@amay To my knowledge, there is no update on this yet.
@deini Do you still work on the PR? What is the status of it?

The README is misleading. It says you can define a time zone when creating a new Job structure, i.e. when creating a job at runtime. This is not possible yet in config.
Any contributions to fix this issue are welcome.

**Comment by c-rack** [comment]
Fixed by merging #121
